woocommerce-gateway-ebanx/includes/notices/class-wc-ebanx-notices-notice.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+if (!defined('ABSPATH')) {
+    exit;
+}
+
+class WC_EBANX_Notices_Notice {
```

Comment:
Where is the comments? All methods are missing comments, we need to comment all of them. Please remember, this project is showing its face to the world 😆